plugins/apache-airflow/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@backstage/plugin-apache-airflow",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.7.6",
+    "@backstage/core-plugin-api": "^0.2.2",
+    "@backstage/theme": "^0.2.14",
+    "@changesets/cli": "^2.14.0",
+    "@material-ui/core": "^4.12.2",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.57",
+    "@microsoft/api-documenter": "^7.13.68",
+    "@microsoft/api-extractor": "^7.18.7",
+    "@microsoft/api-extractor-model": "^7.13.5",
```

Comment:
There are a few dependencies here that aren't in use right? 🤔

Reply:
Yikes, definitely. I've gone ahead and removed unused dependencies. Thanks so much for looking through the code!